### DISCUSSION TITLE
看看克劳德的规则

### DISCUSSION THREAD

**Discussion by TiansuoLi** [viopsa233/AI-testers-Forum#24] [Claude]
它说没有innermonologue这种东西，不知道是在规则限制下装傻还是真没有，要是有和bing一样公开的规则和对话过程就好验证了。🤔

![Screenshot_20230426-020725_Slack](https://user-images.githubusercontent.com/99208095/234365745-5047f1c7-5ba9-49d2-a410-5b4b3d908642.jpg)
![Screenshot_20230426-021019_Slack](https://user-images.githubusercontent.com/99208095/234365768-efad7bce-82c6-4cda-b9f4-92c691f6fc67.jpg)

**Comment by TiansuoLi** [comment]
![Screenshot_20230426-021914_Slack](https://user-images.githubusercontent.com/99208095/234366986-5f8c972c-e5a2-4330-99e7-181d5eb12277.jpg)

![Screenshot_20230426-021921_Slack](https://user-images.githubusercontent.com/99208095/234366994-82e29ec1-9b5e-43ee-99b0-7664ee9c5283.jpg)
![Screenshot_20230426-021927_Slack](https://user-images.githubusercontent.com/99208095/234367005-db575ff4-801a-471e-a521-ba689589eefd.jpg)
![Uploading Screenshot_20230426-021933_Slack.jpg…]()

**Comment by TiansuoLi** [comment]
没问出规则的具体内容
![Screenshot_20230426-024751_Slack](https://user-images.githubusercontent.com/99208095/234373966-66221460-92cf-4dc2-9261-52790999484b.jpg)
![Screenshot_20230426-024758_Slack](https://user-images.githubusercontent.com/99208095/234373993-7a0b6436-dbd8-4e0b-b31a-f5cf5b5bc576.jpg)

**Comment by ghost** [comment]
没有，和ChatGPT一样没有
我用Claude的时候她会偶尔补全出`Human: `，符合官网API文档的输入格式，由此可见Claude的Slack Bot输入输出结构很简单，而Sydney结构复杂是因为Sydney是一个对网络搜索优化的模型，需要同时处理用户输入，搜索引擎输出，内容生成输出，广告输出等等等，也因为这样所以有inner monologue帮助AI思考

**Reply by TiansuoLi** [reply]
> 没有，和ChatGPT一样没有 我用Claude的时候她会偶尔补全出`Human: `，符合官网API文档的输入格式，由此可见Claude的Slack Bot输入输出结构很简单

谢谢老哥，原来真的没有。我想套prompt规则也没套出来。莫非是真的全靠学习过程和微调模型来约束性格和大方向？🤔我还挺喜欢它的说话方式的（

**Reply by ghost** [reply]
ChatGPT也套不出prompt，因为就没有，你可以去OpenAI Playground在线测试GPT的API，就算不填任何系统指令也是有一套规则，输入简单的系统指令也难以覆盖这个规则

**Reply by ghost** [reply]
Claude的官网文档里甚至没有提到系统指令，只有Human和Assistant

**Reply by TiansuoLi** [reply]
> ChatGPT也套不出prompt，因为就没有，你可以去OpenAI Playground在线测试GPT的API，就算不填任何系统指令也是有一套规则，输入简单的系统指令也难以覆盖这个规则

我又了解了一下似乎真的是这样。🤣我之前还以为GPTAPI是有一个比较简单且难以覆盖的底层prompt规则，毕竟从开发角度，也许添加prompt规则比训练更简单易改动嘛（
